assets/js/state/sagas/lastExecutions.js

```diff
@@ -0,0 +1,32 @@
+import { put, call, takeEvery } from 'redux-saga/effects';
+
+import { getLastExecutionByGroupID } from '@lib/api/wanda';
+import {
+  setLastExecutionLoading,
+  setLastExecution,
+  setLastExecutionEmpty,
+  setLastExecutionError,
+} from '@state/lastExecutions';
+
+export function* updateLastExecution({ payload }) {
+  const { groupID } = payload;
+  yield put(setLastExecutionLoading(groupID));
+
+  try {
+    const { data } = yield call(getLastExecutionByGroupID, groupID);
+    yield put(setLastExecution(data));
+  } catch (error) {
+    if (error.response && error.response.status == 404) {
```

Comment:
What is the `404` scenario for?
The moment where there is not any execution for this cluster yet?